flood_api/__main__.py

```diff
@@ -22,12 +25,27 @@ async def lifespan(flood_app: FastAPI):
     version=settings.version,
```

Comment:
`version` and `description` are redundant here, as we redefine them in `openapi.py`. Can be removed